articles/cognitive-services/Computer-vision/concept-describe-images-40.md

```diff
@@ -25,9 +25,11 @@ At this time, image captioning is available in English language only.
 All captions contain gender terms: "man", "woman", "boy" and "girl" by default. You have the option to replace these terms with "person" in your results and receive gender-neutral captions. You can do so by setting the optional API request parameter, **gender-neutral-caption** to `true` in the request URL.
 
 > [!IMPORTANT]
-> Image captioning in Image Analysis 4.0 is only available in the following Azure data center regions at this time: East US, France Central, Korea Central, North Europe, Southeast Asia, West Europe, West US. You must use a Computer Vision resource located in one of these regions to get results from Caption and Dense Captions features.
+> Image captioning in Image Analysis 4.0 is only available in the following Azure data center regions at this time: East US, France Central, Korea Central, North Europe, Southeast Asia, West Europe, West US and East Asia. You must use a Computer Vision resource located in one of these regions to get results from Caption and Dense Captions features.
 >
-> If you have to use a Computer Vision resource outside these regions to generate image captions, please use [Image Analysis 3.2](concept-describing-images.md) which is available in all Computer Vision regions.  
+> If you have to use a Computer Vision resource outside these regions to generate image captions, please use [Image Analysis 3.2](concept-describing-images.md) which is available in all Computer Vision regions.
+> 
+> Image captions are only avalaible in English for now. More languages will be added soon!
```

Comment:
We can't put future promises in the docs

Reply:
Thanks for letting me know. I wasnt aware